doc/level1.rst

```diff
@@ -843,6 +843,13 @@ both programmatically and manually.
   allows implementations to optionally add another human-readable
   explanation.
 
+- The second MIME part's ``Content-Disposition`` SHOULD be
+  ``attachment`` to make it easy for the user to download it as a
+  separate file. If the second MIME part contains HTML, the
```

Comment:
I feel like this second sentence is distracting here. It opens up the whole deal about a "saveable as html" attachment type trickery, which is not trivial and will leave a reader wondering here with no further reference (e.g. search for "html" in the document) on what's going on.

This whole recommendation is sort of a side note (which is also why it's a  "should" rather than "SHOULD" I assume), I think I would either put it in a paragraph of its own that actually describes the idea, or leave it out for brevity. Given how speculative the idea is so far, I would go with the latter for 1.0.

The part about the `Content-Disposition` on the other hand is short and straightforward, it could be included in the previous paragraph right next to the specification of the same part's `Content-Type`.

If anyone +1s this I'll make a PR with this alternative suggestion.

Reply:
I'd be happy to see an alternate PR.